assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -82,10 +88,35 @@ function TextSets() {
     [textSets]
   );
 
-  useEffect(() => {
-    getTextSets().then(setTextSets);
+  const handleSelectedCategory = useCallback((selectedCategory) => {
+    setSelectedCat(selectedCategory);
+    setItemByKey(`${LOCAL_STORAGE_PREFIX.TEXT_SET_SETTINGS}`, {
+      selectedCategory,
+    });
```

Comment:
Since the text pane unmounts and mounts based on its active state, I'm saving off the selected category to local storage so if someone switches tabs, their selected category will persist.